core/src/main/java/org/kohsuke/stapler/export/Property.java

```diff
@@ -168,6 +175,19 @@ private void writeValue(Type expected, Object value, TreePruner pruner, DataWrit
         writeValue(expected,value,pruner,writer,false);
     }
 
+    private void writeBuffered(Type expected, Object value, TreePruner pruner, DataWriter writer) throws IOException {
+        BufferedDataWriter buffer = new BufferedDataWriter();
+        try {
+            writeValue(expected, value, pruner, buffer, true);
+            buffer.finished();
+        } catch (IOException x) {
+            if (x.getCause() instanceof InvocationTargetException) {
+                LOGGER.log(Level.WARNING, "skipping export of " + value, x);
```

Comment:
Several logged exceptions per API request seems a bit much to me, especially given how common the problem of unexportable data seems to me. I'd expect the frequency to be about as annoying as what https://github.com/stapler/stapler/pull/48 fixed.